packages/volto/cypress/tests/core/content/a11y.js

```diff
@@ -0,0 +1,100 @@
+describe('Accessibility Tests Content Types', () => {
+  beforeEach(() => {
+    cy.autologin();
+    cy.visit('/');
+    cy.injectAxe(); // make sure axe is available on the page
+  });
+
+  it('Event has no a11y violations', () => {
+    cy.get('#toolbar-add').click();
```

Comment:
@ana-oprea add more dates and information to the event so that in the demo you have the right hand side with the event information

Reply:
@ichim-david when an event is created, the start date and end date (defaulting to today's date) are automatically added and displayed on the right-hand side along with the event information. This is how the right hand side looks in the test
![image](https://github.com/user-attachments/assets/aca97754-a886-4faf-b808-cffbe53c7e6d)